src/App.jsx

```diff
@@ -1,7 +1,8 @@
 var React = require('react');
+var ReactDOM = require('react-dom');
 var Locations = require('./components/Locations.jsx');
```

Comment:
Better to not include the file extension
`var Locations = require('./components/Locations');` should work